sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -1445,4 +1445,10 @@ private[spark] object QueryCompilationErrors {
   def operationNotSupportPartitioningError(operation: String): Throwable = {
     new AnalysisException(s"'$operation' does not support partitioning")
   }
+
+  def mixedRefInAggFunc(a: Expression): Throwable = {
```

Comment:
nit: `mixedRefInAggFunc` -> `mixedRefsInAggFunc `

Reply:
We can tighten the type here:`a: Expression` -> `a: AggregateExpression`